tapdb/universe_perf_long_test.go

```diff
@@ -6,7 +6,7 @@ package tapdb
 const longTestScale = 5
 
 var (
-	numAssets         = 100 * longTestScale
-	numLeavesPerTree  = 300 * longTestScale
-	numQueries        = 100 * longTestScale
+	numAssets        = 100 * longTestScale
+	numLeavesPerTree = 300 * longTestScale
+	numQueries       = 100 * longTestScale
```

Comment:
Should the test timeout be a bit longer for the `_long_test.go` or do you think the 5 minute timeout is ok?